test/integration/shoots/maintenance/maintenance.go

```diff
@@ -0,0 +1,338 @@
+
+		f.Beta().Serial().CIt("Kubernetes version should be updated: force update minor version", func(ctx context.Context) {
+			defer func() {
+				// make sure to remove expiration date from cloud profile after test
+				err = TryUpdateCloudProfileForKubernetesVersionMaintenance(ctx, f.GardenClient.Client(), f.Shoot, testKubernetesVersionHighestPatchLowMinor.Version, &expirationDateInThePast, &deprecatedClassification)
+				gomega.Expect(err).To(gomega.BeNil())
+				f.Logger.Infof("Cleaned expiration date on kubernetes version from Cloud Profile '%s'", testShootCloudProfile.Name)
+			}()
+
+			// set the shoots Kubernetes version to be the highest patch version of the minor version
+			f.Shoot.Spec.Kubernetes.Version = testKubernetesVersionHighestPatchLowMinor.Version
+			err = TryUpdateShootForKubernetesMaintenance(ctx, f.GardenClient.Client(), f.Shoot)
+			gomega.Expect(err).To(gomega.BeNil())
+
+			// let Shoot's Kubernetes version expire
+			err = TryUpdateCloudProfileForKubernetesVersionMaintenance(ctx, f.GardenClient.Client(), f.Shoot, testKubernetesVersionHighestPatchLowMinor.Version, &expirationDateInThePast, &deprecatedClassification)
+			gomega.Expect(err).To(gomega.BeNil())
+
+			err = StartShootMaintenance(ctx, f.GardenClient.Client(), f.Shoot)
+			gomega.Expect(err).To(gomega.BeNil())
+
+			// expect shoot to have updated to latest patch version of next minor version
+			err = WaitForExpectedKubernetesVersionMaintenance(ctx, f.Logger, f.GardenClient.Client(), f.Shoot, testKubernetesVersionHighestPatchConsecutiveMinor.Version, true, time.Now().Add(time.Second*20))
+			gomega.Expect(err).To(gomega.BeNil())
+		}, waitForCreateDeleteTimeout)
+	})
+})
+
+func prepareCloudProfile(ctx context.Context, profile gardencorev1beta1.CloudProfile, shoot *gardencorev1beta1.Shoot) {
+	// setup cloud profile for integration test
+	machineImageName := shoot.Spec.Provider.Workers[0].Machine.Image.Name
+	found, image, err := helper.DetermineMachineImageForName(&profile, machineImageName)
+	gomega.Expect(err).To(gomega.BeNil())
+	gomega.Expect(found).To(gomega.Equal(true))
+
+	imageVersions := append(image.Versions, gardencorev1beta1.ExpirableVersion{Version: testMachineImageVersion, Classification: &deprecatedClassification})
+	updatedCloudProfileImages, err := helper.SetMachineImageVersionsToMachineImage(profile.Spec.MachineImages, machineImageName, imageVersions)
+	gomega.Expect(err).To(gomega.BeNil())
+	// need one image in Cloud Profile to be updated with one additional version
+	profile.Spec.MachineImages = updatedCloudProfileImages
+
+	// add  test kubernetes versions (one low patch version, one high patch version)
+	profile.Spec.Kubernetes.Versions = append(profile.Spec.Kubernetes.Versions, testKubernetesVersionLowPatchLowMinor, testKubernetesVersionHighestPatchLowMinor, testKubernetesVersionLowPatchConsecutiveMinor, testKubernetesVersionHighestPatchConsecutiveMinor)
+	err = f.GardenClient.Client().Update(ctx, &profile)
```

Comment:
Same question here...

Reply:
Unsure why this is related?

Reply:
Well, you're updating the CloudProfile of a gardener system here. So do we have a test instance for those tests?

Reply:
I do not think so. That depends how the test machinery is set up.  Thats why it is important to remove these changes and adding only low versions that cannot be defaulted to. @schrodit